lblod/ember-rdfa-editor-lblod-plugins
# GN-4445 Move buttons blue for template comments

### Overview
Makes the moveUp and moveDown buttons for template comments blue.

##### connected issues and PRs:
https://binnenland.atlassian.net/browse/GN-4447


### How to test/reproduce
click in empty document, in insert press "insert template comment"
select inside template comment and check buttons on side if they are blue.
